app/models/instrument_for_cart.rb

```diff
@@ -0,0 +1,18 @@
+class InstrumentForCart < ProductForCart
+
+  def purchasable_by?(acting_user, session_user)
+    super.tap do |success|
```

Comment:
I feel like the `super.tap` is a weird construct here. It'll always return the value of `super`, so even if we add an error inside the block, the method will still return `true` if it was successful in the superclass.

Reply:
With the refactoring of `ProductForCart#purchasable_by?` into separate methods that I did in 166039e37, I think I addressed this as well. What do you think?